src/chapters/classes/srp.rst

```diff
@@ -0,0 +1,33 @@
+Single Responsibility Principle
+-------------------------------
+
+As we wrap up our whirlwind tour of encapsulation, we want you think a
```

Comment:
does that mean that the last few sections should be nested underneath encapsulation? also the header above this line should use ===

Reply:
That should say "classes". Fixed! Thanks!